prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -401,7 +404,10 @@ class ActionHistory : Logging {
         trackItemLineages(report)
 
         // batch queue messages are added by the batchDecider, not ActionHistory
-        if (event.eventAction != Event.EventAction.BATCH)
```

Comment:
doc here should be updated to explain ROUTE and TRANSLATE being redirected

Reply:
Updated.